package.json

```
@@ -15,7 +15,7 @@
     "lint": "turbo run lint",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "release": "propel-release",
-    "test": "echo 'No tests yet'",
+    "test": "yarn workspaces foreach run test",
```

Will may also do it with `turbo`
```suggestion
    "test": "turbo run test",
```

if we add the following into [turbo.json](https://github.com/propeldata/ui-kit/blob/6504d53c196668a0b8206c489d3934faf00b5552/turbo.json#L11) config:
```
    "test": {
      "dependsOn": ["build"]
    },
```

---

Right, turbo will work a lot better.
I think it's not necessary to depend on build though, since the tests are not using the bundle but the `.tsx` file directly.
I'm changing this! 👍